mixer/pkg/config/crd/store.go

```diff
@@ -145,13 +153,41 @@ func (s *Store) Init(kinds []string) error {
 	s.caches = make(map[string]cache.Store, len(kinds))
 	s.informers = make(map[string]cache.SharedInformer, len(kinds))
 	remaining := s.checkAndCreateCaches(d, lwBuilder, kinds)
+	timeout := time.After(s.retryTimeout)
+	tick := time.Tick(s.retryInterval)
+	stopRetry := false
+	if tick != nil {
```

Comment:
when will tick be `nil` it is initialized 2 lines above that.

Reply:
Right, it is not necessary. Initially I thought retryInterval should also be configurable and tick will be nil if interval is configured as 0. But now it is not configurable so it should be safe to assume tick cannot be nil.